setup.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+
+"""
+Exports the appropriate symbols from the module
+
+MIT License
+
+Copyright (c) 2020
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+"""
+
+
+##============================================================================
+##  Imports
+##============================================================================
+import os, sys, subprocess, setuptools, shutil
+
+
+##============================================================================
+##  Use setuptools normally.
+##============================================================================
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+setuptools.setup(
+    name="pyvirtualbench",
+    version="1.0.1",
+    author="armstrap",
+    author_email="charles@armstrap.org",
```

Comment:
For author_email, can we use armstrap@users.noreply.github.com instead?  Thanks

Reply:
Yup.  Fixed.

Reply:
Thanks!